boswatch.py

```diff
@@ -156,7 +156,9 @@
 		logging.debug("BOSWatch given arguments")
 		if args.test:
 			logging.debug(" - Test-Mode!")
-
+			globalVars.testvariable = 1
+		else:
+			globalVars.testvariable = 0
```

Comment:
Was ist das? Wofür wird das benötigt?

Reply:
Das ist eine Testvariable, mit der ich unter anderem in meinem Testbetrieb geschaut hab, ob Boswatch im Testmodus lief, somit konnte ich die Probealarmierungen abfangen und den Start anderer Plugins unterbinden, kann aber gern weg.